spaces/domains.json

```
@@ -1,4 +1,5 @@
 {
+  "governance.cyberestate.io": "cyberrealestate.eth"
```

A comma is missing at the end of the line of your sub-domain.
```suggestion
  "governance.cyberestate.io": "cyberrealestate.eth",
```

---

oh ok, I fixed it, I can't believe I missed that. Thank you for your help I appreciate it.